tests/positive/partitions/mixed.go

```diff
@@ -35,22 +35,22 @@ func Match1Recreate1Delete1Create1() types.Test {
 				Label:    "important-data",
 				Number:   1,
 				Length:   65536,
-				TypeGUID: "B921B045-1DF0-41C3-AF44-4C6F280D3FAE",
-				GUID:     "8A7A6E26-5E8F-4CCA-A654-46215D4696AC",
+				TypeGUID: "$uuid0",
+				GUID:     "$uuid1",
 			},
 			{
 				Label:    "ephemeral-data",
 				Number:   2,
 				Length:   65536,
-				TypeGUID: "CA7D7CCB-63ED-4C53-861C-1742536059CC",
-				GUID:     "B921B045-1DF0-41C3-AF44-4C6F280D3FAE",
```

Comment:
we shouldn't have duplicate guids ever. This was a bug beforehand, but we should fix it in this PR

Reply:
Fixed this and all above comments related to fixing UUID variables.